### ISSUE TITLE
Render natural=bare_rock earlier

### ISSUE THREAD

**Issue by mfbehrens99** [gravitystorm/openstreetmap-carto#4190]
Especially when looking at some mountain ranges from far (>=12) they look very blank. A reason is that bare rock is not rendered on zomm levels 12, 11 and 10.

**Comment by imagico** [comment]
natural=bare_rock is rendered starting zoom level 5 - like the other landcover types.  It is very badly visible at the low zoom levels due to the color fading used - see #3936.  What starts at z13 is the pattern rendering. That is a conscious design choice for almost all patterns (see #4072).

Closing as either invalid or duplicate of #3936 - but it makes a very valid point for #3936 even after #3952.  See:

https://www.openstreetmap.org/#map=8/67.902/91.794
